### ISSUE TITLE
Set learning rate at specify layer

### ISSUE THREAD

**Issue by eric612** [pjreddie/darknet#1201]
I have a question , if I set learning rate at each convolution layer (like below) , is it effective ?

[convolutional]
batch_normalize=1
filters=128
size=3
stride=2
pad=1
activation=leaky
learning_rate=0.0001

[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky
learning_rate=0.0001

**Comment by xiuyangleiasp** [comment]
it is not support in current code. 
but you can implement it in the function "update_network"

**Comment by eric612** [comment]
Ok , thanks a lot
